rancher2/schema_node_template_hetzner.go

```diff
@@ -37,6 +38,11 @@ func hetznerConfigFields() map[string]*schema.Schema {
 			Default:     "ubuntu-18.04",
 			Description: "Hetzner Cloud server image",
 		},
+		"server_labels": {
+			Type:        schema.TypeMap,
+			Optional:    true,
+			Description: "Comma-separated list of labels which should be assigned to the server",
```

Comment:
Typo at the argument description. The `server_labels` argument is now a map, it can't be defined as a `Comma-separated list of labels..`. https://www.terraform.io/language/expressions/types#maps-objects